components/gcp/bigquery/query/component.yaml

```diff
@@ -33,20 +34,23 @@ inputs:
     default: '' 
     type: String
   - name: output_gcs_path
-    description: 'The GCS blob path to dump the query results to.'
+    description: 'The path to the Cloud Storage bucket to store the query output.'
```

Comment:
To be honest, the word "bucket" is confusing here. You've probably meant "The path of GCS directory" or "The path of GCS file".

Reply:
This is a suggested change from tech writer. I'd like to follow their suggestion to make it consistent with other AIHub docs.

Reply:
OK.
We should probably still tell them that technically, a bucket is `my-bucket` while `gs://my-bucket/some/dir/` is a GCS directory path, not a bucket.